docker/travis.tests.sh

```diff
@@ -19,7 +19,7 @@ cd ${dagmc_build_dir}/bld
 # If this is not a pull request, get files needed for regression tests
 if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
   cd src/make_watertight/tests
-  wget ${MW_REG_TEST_MODELS_URL} -O mw_reg_test_files.tar.gz -o wget.out
+  wget ${MW_REG_TEST_MODELS_URL} -O mw_reg_test_files.tar.gz
```

Comment:
The comment suggested streaming these to `stderr` and `stdout` - do we do this now?  Or is that default behavior?

Reply:
my understanding of their comment was the default streams them to `stderr`